parser/Cargo.toml

```diff
@@ -4,3 +4,9 @@ edition = "2018"
 license = "MIT OR Apache-2.0"
 name = "defmt-parser"
 version = "0.1.0"
+
+[features]
+unstable = []
+
+[package.metadata.docs.rs]
+rustdoc-args = [ "--cfg docsrs" ]
```

Comment:
Oh, I think we'll need an additional line in the docs.rs configuration:

```toml
features = ["unstable"]
```

Otherwise the crates will be empty when the docs are generated

Reply:
I agree with Jonas. All the Cargo.toml that have an `unstable` feature should have that line under the `[docs.rs]` section. (Though I guess not showing any documentation at all may be another way to say that the API is not meant to be used..)

Reply:
yeah I thought that was the goal here :D will change in a minute